CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @alphagov/di-devplatform
```

Comment:
hmm, not picking it up
let me double check the team name

Reply:
should be @alphagov/di-dev-platform

Reply:
I think the team needs to be public but I don't know how to see that